charts/flink-operator/templates/clusterroles.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-flink-resources-edit
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups: [ flinkoperator.k8s.io ]
+    resources: [ "*" ]
+    verbs: [ create, delete, deletecollection, patch, update]
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think you may be able to downgrade this to being just a `Role` instead of a `ClusterRole` as well, but I would need to check more carefully...

Reply:
This makes sense. Looks like the diff is just scope (namespace vs cluster), but I have verified `ClusterRole`, will verify Role as well.